diff --git a/configure.ac b/configure.ac
index 261fb06..cc5d658 100644
--- a/configure.ac
+++ b/configure.ac
@@ -341,7 +341,14 @@ AC_CHECK_HEADER([stdint.h],
 		[EINA_CONFIGURE_HAVE_STDINT_H="#define EINA_HAVE_STDINT_H"])
 AC_SUBST(EINA_CONFIGURE_HAVE_STDINT_H)
 
-AC_CHECK_SIZEOF(wchar_t)
+AC_TRY_COMPILE([#include <wchar.h> typedef wchar_t w;],,
+						    ac_cv_have_wchar_t=yes, ac_cv_have_wchar_t=no)
+
+ac_cv_sizeof_wchar_t=0
+if test "x${ac_cv_have_wchar_t}" = "xyes" ; then
+	AC_CHECK_SIZEOF(wchar_t, 4, [wchar.h])
+fi
+
 EINA_SIZEOF_WCHAR_T=$ac_cv_sizeof_wchar_t
 AC_SUBST(EINA_SIZEOF_WCHAR_T)
 
